knexfile.js

```diff
@@ -1,14 +1,13 @@
 // Update with your config settings.
-
 module.exports = {
 
   development: {
     client: 'pg',
     connection: {
       host: 'localhost',
-      user: 'euniceem1007',
-      password: 'password',
-      database: 'react-kanban'
+      user: process.env.POSTGRES_USER,
```

Comment:
This looks good with no sensitive information committed. Ask an instructor and we can show you how to condense it down to one single object 😃